app/dashboard/notifications.py

```diff
@@ -36,6 +36,14 @@
 from slackclient import SlackClient
 
 
+# Be VERY CAREFUL when changing this setting.  You don't want to accidently
+# send a bunch of github notifications :)
+# See https://medium.com/gitcoin/github-notification-spam-post-mortem-e1ed200936e
+class CowardlyGitcoinBotException(Exception):
+    pass
+cowardly_bot_exception = CowardlyGitcoinBotException('Cowardly ')
```

Comment:
Quick question. Just for the sake of better readability and best practices. Should we use a global variable like this or should we raise exception with the object instantiation at every check? Because doesn't this way this object get instantiated even if it might not be needed in the code?
@mbeacom views?